aries-site/src/themes/aries.js

```diff
@@ -209,6 +210,27 @@ export const aries = deepMerge(hpe, {
       size: 'medium',
     },
   },
+  pagination: {
+    button: {
+      border: {
+        radius: '100px',
```

Comment:
Should these be 100%?

```suggestion
        radius: '100%',
```

Reply:
I went with 100px because I was getting strange results with 100% when I was doing the other button work. 100% seemed to "overround". It's less of an issue for pagination, but still subtley overrounded I feel. See screenshots below and let me know what you think:

primary button with radius 100%
<img width="128" alt="Screen Shot 2023-01-27 at 1 06 15 PM" src="https://user-images.githubusercontent.com/12522275/215199141-5c1fe62d-5069-4b77-aacc-4d82b06f8d36.png">

pagination button with radius 100% (less likely case because the label will rarely be this long)
<img width="788" alt="Screen Shot 2023-01-27 at 1 05 08 PM" src="https://user-images.githubusercontent.com/12522275/215199209-48a48283-fef0-4bb8-bac5-8e449c96194f.png">

pagination button with radius 100px (feels uniform)
<img width="802" alt="Screen Shot 2023-01-27 at 1 04 52 PM" src="https://user-images.githubusercontent.com/12522275/215199382-a45856ae-a2d7-4be9-a128-8d2c8a33d3e9.png">